airbyte-webapp/src/config/index.ts

```diff
@@ -55,10 +54,7 @@ const config: {
     enableStorytime: window.PAPERCUPS_STORYTIME !== "disabled",
   },
   version: window.AIRBYTE_VERSION,
-  apiUrl:
-    window.API_URL ||
-    process.env.REACT_APP_API_URL ||
-    `${window.location.protocol}//${window.location.hostname}:8001/api/v1/`,
+  apiUrl: "/api/v1/",
```

Comment:
I do not think this is correct: it still possible to host webapp separately from the server, so we shouldn't hardcode URL and there still should be a way to specify different server url.